user_manual/android_app.rst

```diff
@@ -320,6 +376,37 @@ The bottom section of the *"Settings"* screen has links to:
 - Feedback 
 - The version number
 
+.. _pattern_and_fingerprint_lock_label:
+
+Pattern Lock and Fingerprint Lock
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. versionadded:: 2.7.0
+
+In addition to the Passcode Lock and Pins, you also have the ability to use both a pattern and a fingerprint lock to protect access to your ownCloud app and its data.
```

Comment:
What do you mean with Pins here?

Reply:
That's a reference to the login pins. I'll update that line to make it more specific.